### ISSUE TITLE
[feature request] please add a launcher gesture

### ISSUE THREAD

**Issue by revuwa** [TheAndroidMaster/WearLocker#3]
Thanks for your great app and for sharing the source code with the world.

Android wear 2.0 seems to require a physical button, so some (button-less) devices with ported wear 2.0, like the LG G Watch for example (https://forum.xda-developers.com/g-watch/development/port-lenok2dory-android-wear-lg-g-watch-t3624441) leaves the users with the struggle that they have to search for alternative ways to access the apps panel. By the way, there is a shaking gesture for it, but this doesn't work properly...

Long story short. I tried your app on my device (with ported wear 2.0) and it works perfect. So I thought it would be very cool, if I could open the apps panel, (that action usually the physical button does) with a gesture inside of your app. Hope you don't get me wrong, I think this feature could be very useful for all people (which want easy and fast access to this trigger, or just want to avoid pressing the physical button that much, etc.), not just these with wear 2.0 port.

For better understanding, I made a picture what came into my mind:
![request](https://user-images.githubusercontent.com/7300276/28135760-9c2abbb8-6747-11e7-9fff-ca6fe81e68ea.png)
So the launcher (or better name) gesture point could be disabled by default, but could be set to the gesture set of the unlock gesture, for example: swipe right.

In this example I could unlock my watch by double tapping on the display, but also could unlock & launch the apps panel by swiping right.

What do you think?

**Comment by fennifith** [comment]
I agree that this would definitely be a great addition to this app. Forcing users to use a physical button or shake gesture instead of the old method of opening the app drawer was one of the changes in AW2 that I wasn't particularly fond of. However, I have already tried to implement a similar feature before, and I couldn't find any ways to open the launcher/app drawer programmatically unless I were to build a "fake" replica into the app and open that instead, which seems a bit wasteful and is probably not the best solution IMO.

**Comment by revuwa** [comment]
Thanks for your response. I found this project: https://github.com/KonradIT/wearbttn just right now. Seems root rights are required for opening the 'app drawer', which is also not the best solution, (but propbably the only one). Rats! Hope somebody finds a rootless solution (if this is possible at all) soon.
